static/templates/settings/edit_bot.hbs

```diff
@@ -2,22 +2,22 @@
   data-email="{{bot.email}}" data-type="{{bot.bot_type}}">
     <div class="bot_edit_errors alert alert-error hide"></div>
     <div>
-        <div>
-            <label>{{t "Bot email" }}</label>
-            <div class="edit_bot_email">{{bot.email}}</div>
-        </div>
         <div class="edit-bot-form-box">
+            <div class="input-group edit-bot-name">
+                <label for="edit_bot_name">{{t "Full name" }}</label>
+                <input id="edit_bot_name" type="text" name="bot_name" class="edit_bot_name required" value="{{bot.full_name}}" maxlength=50 />
+                <div><label for="edit_bot_name" generated="true" class="text-error"></label></div>
+            </div>
+            <div class="input-group">
+                <label for="bot_email">{{t "Bot email" }}</label>
```

Comment:
@alya Can we also change the heading here to be `Email` like we do in other modal?

Reply:
Yes, definitely -- good catch!